### PR TITLE
Update shared_capabilities.es.md

### PR DESCRIPTION
**Pull request by AlexAndradeNet**
Translation to Spanish.

I've agreed the CLA

### CODE REVIEW

File: docs_source_files/content/driver_idiosyncrasies/shared_capabilities.es.md [hunk 1]
```diff
@@ -3,13 +3,7 @@ title: "Capacidades compartidas"
...
 
-{{% notice info %}}
-<i class="fas fa-language"></i> Page being translated from 
-English to Spanish. Do you speak Spanish? Help us to translate
-it by sending us pull requests!
-{{% /notice %}}
-
-### pageLoadStrategy
+## pageLoadStrategy
 When navigating to a new page via URL, by default Selenium will wait
```
Comment (diemol) [inline docs_source_files/content/driver_idiosyncrasies/shared_capabilities.es.md]: There is still some text in English, can you please remove it?

**Review by diemol — CHANGES_REQUESTED** [review]

**Review by AlexAndradeNet — COMMENTED** [review]
Was removed some paragraphs that still remained in English

File: docs_source_files/content/driver_idiosyncrasies/shared_capabilities.es.md (hunk 1 above)
Comment (AlexAndradeNet) [reply docs_source_files/content/driver_idiosyncrasies/shared_capabilities.es.md]: Was removed some paragraphs that still remained in English

**Review by diemol — APPROVED** [review]
Thank you @AlexAndradeSan!

**Pull request merged**